ipywidgets/widgets/widget.py

```diff
@@ -483,7 +483,7 @@ def _compare(self, a, b):
         else:
             return a == b
 
-    def _compare_buffers(self, a, b, key):
+    def _buffer_list_equal(self, a, b):
```

Comment:
The current implementation works for me, but if someone uses a serializer/deserialize set that might introduce noise when data is passed through a loop (`a != serializer(deserialize(a))`), e.g. floating point inaccuracies, they will want to override this comparison.

It is still possible to do that by overriding the `_should_send_property` function, but it will be more brittle as it needs to take into account the internals of that method (e.g. `_property_lock`), making it more brittle.